index.js

```diff
@@ -25,7 +25,11 @@ module.exports = function (grunt, opts) {
 
 	multimatch(names, pattern).forEach(function (pkgName) {
 		if (opts.requireResolution === true) {
-			grunt.loadTasks(resolvePkg(path.join(pkgName, 'tasks')));
+			try {
+				grunt.loadTasks(resolvePkg(path.join(pkgName, 'tasks')));
+			} catch (e) {
+				// package can't be loaded
```

Comment:
Should it really just silence the error?

How about logging `grunt.verbose('Task ' + pkgName + ' couldn\'t be found');`?